enhancements/okd-on-fedora-coreos-in-prow.md

```diff
@@ -78,7 +78,7 @@ This is a proposal to:
 ### Graduation Criteria
 
 - [x] [OKD repository](https://github.com/openshift/okd/) exists for community issue and bug triage, and development tracking.
-- [x] The UBI-based container images for all components in the release payload are shared between OKD and OCP for testing in Prow (with a few exceptions, see below)
+- [ ] Container images for all components used to build or in the release payload are built atop Centos Stream base images most closely matching the majority of OCP images (as of Feb 2024, CS9) and do not contain any content from RHEL builds of any packages. Base images and builders will be maintained jointly by OKD and ART teams. OCP Engineering teams will be responsible for addressing bugs reported against OKD in their individual images but should otherwise have minimal steady state image maintenance burden, adding of net new packages will be required of the team adding those packages however.
```

Comment:
>OCP Engineering teams will be responsible for addressing bugs reported against OKD

Who's responsible for triaging OKD bugs - both in Jira and in github issues?

Reply:
>do not contain any content from RHEL builds of any packages

How are we planning to enforce that? There is nothing preventing devs from using internal RPM repos

Reply:
I think Github would need to be triaged first by OKD team/volunteers and if necessary OCPBUGS filed. They're just OCP bugs then so they'd be triaged by all the normal teams.

As to enforcement of no RHEL content that will depend on the implementation path chosen.